sparksteps/defaults.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+"""
+Application CLI defaults
+"""
+DEFAULT_JOBFLOW_ROLE = 'EMR_EC2_DefaultRole'
+DEFAULT_APP_LIST = ['Hadoop', 'Spark']
```

Comment:
Should we just put these at the top of `cluster.py` and import from there? Not a huge fan of constants/defaults/utils type of files.

Reply:
hmm, yeah a separate file might be overkill at the moment, was thinking of making it more future proof but can break it up into a separate file later when more constants are expressed. Sounds good.